include/aws/checksums/exports.h

```diff
@@ -4,8 +4,7 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0.
  */
-#if defined(USE_WINDOWS_DLL_SEMANTICS) || defined(WIN32)
-
+#if defined(USE_WINDOWS_DLL_SEMANTICS) || defined(_WIN32)
 #    ifdef USE_IMPORT_EXPORT
```

Comment:
AWS_CHECKSUMS_USE_IMPORT_EXPORT

Reply:
Not sure it should be this name, because this name doesn't exist anywhere.
Also in export.h of aws-c-common there is a line
```
#    if ((__GNUC__ >= 4) || defined(__clang__)) && defined(AWS_COMMON_USE_IMPORT_EXPORT) && defined(AWS_COMMON_EXPORTS)
```
Where I didn't found the set up of these two macros in code base. `AWS_COMMON_USE_IMPORT_EXPORT ` and `AWS_COMMON_EXPORTS`

Directly copy this line to aws-checksums's export.h and rename it with AWS_CHECKSUMS_ prefix won't work as well. Becuase it's not defined anywhere. So just removed it to enable visibility

Reply:
AWS_CHECKSUMS_USE_IMPORT_EXPORT